setup.cfg

```diff
@@ -0,0 +1,15 @@
+[flake8]
+# References:
+# http://flake8.readthedocs.org/en/latest/config.html
+# http://flake8.readthedocs.org/en/latest/warnings.html#error-codes
+#
+# Style checks turned on:
+#   F - all pyflake errors
+#   E101 - indentation contains mixed spaces and tabs
+#   E111 - indentation is not a multiple of four
+#   E501 - line too long (see max-line-length)
+
+exclude = bokeh/models/__init__.py, bokeh/models/widgets/__init__.py
+ignore = E,W
+select = F,E101,E111,E501
+max-line-length = 120
```

Comment:
This is where we can configure flake8. This turns off all style errors and warnings, and then turns on just the ones we want. I think E101 and E111 make sense. E501 can be turned off if majority prefers that; I'm just showing what we can do (it now allows lines up to 120 chars)